src/main/resources/META-INF/rust-core.xml

```diff
@@ -1009,10 +1009,22 @@
 
     </extensions>
 
+    <extensions defaultExtensionNs="org.rust">
+        <!-- SDK Flavors -->
+        <sdkFlavor implementation="org.rust.ide.sdk.flavors.RustupSdkFlavor"/>
```

Comment:
Let's add id for each flavor. It allows us and external developers properly configure order of suggestions